src/compiler/ast/common/Node.ts

```diff
@@ -1299,15 +1299,16 @@ export class Node<NodeType extends ts.Node = ts.Node> {
     }
 
     /**
-     * Gets the line number at the start of the node.
+     * Gets the line number (1-indexed) of the start of the node in the *full* text of its source file. 
```

Comment:
Thanks! This is an improvement, but let's remove the emphasis (as nothing makes sense other than full text) and use "at" for both of these.

```suggestion
     * Gets the line number (1-indexed) at the start of the node in the full text of its source file. 
```